calico_containers/pycalico/datastore.py

```diff
@@ -434,6 +434,8 @@ def create_profile(self, name):
         Note this will clobber any existing profile with this name.
 
         :param name: Unique string name for the profile.
+        :param rules: The rules to set on the profile
+        :type rules Rules
```

Comment:
This docstring needs to explain that `rules` is optional and what happens if it isn't specified.